app/concepts/Payment/wrapper/use_case/buy.rb

```diff
@@ -17,15 +17,14 @@ def call(id,password, ticket_params)
         end
     
     private
-
-    def validate?(id,password, left_Repository, right_Repository, seat, params)
+    #checks if email=password not blank and seat can be taken
```

Comment:
Jeśli zostawiasz takie komentarze w kodzie to jest to sygnał, że kod nie jest zrozumiały. Zastanów się nad wydzieleniem z tej metody odpowiedzialności do innych metod i odpowiednim nazywaniem metod i zmiennych.

Reply:
@w0st 
Czyli komentarze mają być jeszcze krótsze? 
Coś na zasadzie #walidacja płatności, ale w wtedy komentarza nie trzeba dawać, bo sama nazwa mówi o co chodzi.

Czy mógłbyś mi pokazać przykład dobrze napisanego komentarza?

Reply:
https://github.com/KMosc/APITRY/pull/17#discussion_r649822115

Powtórze jeszcze raz: Na własne życzenie utrudniasz czytelnikom zrozumienie, co się dzieje w tej metodzie. Zamiast pisać komentarze poświeciłbym ten czas na przepisanie tego kodu by był czytelniejszy. Nazwy metod  i zmiennych powinny same mówić co się dzieje z kodem. Jeśli tak nie jest to jest to sygnał, że nazwy są źle dobrane.